### PR TITLE
Need some work to do, but I am on the right track

### PR DESCRIPTION
**Pull request by M-ram224**

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -7,18 +7,54 @@ var secretCodeWord1 = "lackadaisical"
...
+  // map over the str to change the str val
+  mappedCode = splitStr.map((valA) => {
+    // create if statement
+    // need to change the vowels to a number
+    // let valA = "a"
+    // let valE = "e"
+    // let valI = "i"
+    // let valO = "o"
+    
+    if (valA === 4) {
```
Comment (minachuong) [inline code-challenges.js:23]: Looks like you're practically there. Ideally, codedMessage() should be able to called with any string and always yield the correct result without commenting/uncommenting code. 

Also consider how you could store this mapping between a letter to a number. Think about data types :) Feel free to ask me about this.

File: code-challenges.js [hunk 2]
```diff
@@ -31,6 +67,14 @@ var hand3 = [5, 5, 5, 5, 4]
 var hand4 = [3, 5, 3, 5, 3]
 // Expected output: true
 
+// function will take an array of nums
+const full_house (arr) => {
+    // identify the particular indexes numbers in the array
```
Comment (minachuong) [inline code-challenges.js:72]: good strategy. need a little more detail on what "identify the particular indexes numbers" is trying to accompolish

File: ruby_challenges.rb [hunk 3]
```diff
@@ -2,44 +2,80 @@
...
-
-
-
-
-# Expected output example: 'The Trek bike has 2 wheels and a 168cm frame.'
-
+class Bike
+    def initialize (model, wheels, frame_size, bell, speed)
+        @model = model
+        @wheels = "2 wheels"
```
Comment (minachuong) [inline ruby_challenges.rb:8]: consider the data type for @wheels. Should it be string or number? Why? And depending on that answer what should the default value be?

**Review by minachuong — APPROVED** [review]
JS challenges: good strategies. Consider using different data types to optimize your solution
Interview questions: awesome!

File: code-challenges.js (hunk 1 above)
Comment (M-ram224) [reply code-challenges.js:23]: Yeah, I was looking at many different thoughts, I had .map as my original idea then switched it. What other data points would be used?

**Pull request merged**